cv_dv_utils/LICENSE

```diff
@@ -0,0 +1,202 @@
+         Apache License
```

Comment:
This file is redundant as all files at or below this directory must be covered by one of the open-source accepted by OpenHW: Solderpad-2.x, Apache-2.x or EPL.

Reply:
Update done: File removed